mip/term.go

```diff
@@ -0,0 +1,15 @@
+package mip
+
+// Term is the building block of a constraint and an objective. A term consist
+// of a coefficient and a var and should be interpreted as the product
+// of coefficient and the var in the context of the constraint or
+// objective.
+type Term interface {
+	// Coefficient return the coefficient value of the invoking term.
```

Comment:
```suggestion
	// Coefficient returns the coefficient value of the invoking term.
```

Really minor, but sounds nicer to me.